apps/prairielearn/src/pages/instructorFileEditor/instructorFileEditor.ejs

```diff
@@ -147,13 +147,15 @@
                             </span>
                         </div>
                         <div class="col-auto collapse <% if ((!fileEdit.failedPush) && (!fileEdit.alertChoice)) { %>show<% } %>" id="buttons-<%= fileEdit.uuid %>">
-                            <a id="help-<%= fileEdit.uuid %>-button"
-                               class="btn btn-light btn-sm"
-                               data-toggle="collapse" href="#help-<%= fileEdit.uuid %>"
-                               role="button" aria-expanded="false">
+                            <button
+                                id="help-<%= fileEdit.uuid %>-button"
+                                class="btn btn-light btn-sm"
+                                data-toggle="collapse"
+                                data-target="#help-<%= fileEdit.uuid %>"
+                                aria-expanded="false">
```

Comment:
```suggestion
                            <button
                                type="button"
                                id="help-<%= fileEdit.uuid %>-button"
                                class="btn btn-light btn-sm"
                                data-toggle="collapse"
                                data-target="#help-<%= fileEdit.uuid %>"
                                aria-expanded="false">
```
This button is in a form, so I think it's good to be explicit.

Reply:
Good call, I added a few more missing `type="button"` attributes in af60b6a.